### PR TITLE
NAS-112510 / 21.10 / Filter svclb from pod selector

### PR DESCRIPTION
**Pull request by Ornias1993**
This PR removes all "Service Loadbalancer" pods from the log and shell selector.
These pods are created by the K3S ServiceLoadbalancer and their logs have no value for the user.

They also lead to significant confusion, with users submitting the wrong logs.

**Before:**
---
image

image

**After:**
---
image

image

### CODE REVIEW

File: src/app/pages/applications/chart-releases/chart-releases.component.ts
Comment (undsoft) [inline src/app/pages/applications/chart-releases/chart-releases.component.ts:506]: Let's add a comment explaining what this does.

File: src/app/pages/applications/chart-releases/chart-releases.component.ts
Comment (undsoft) [inline src/app/pages/applications/chart-releases/chart-releases.component.ts:505]: You can probably just do
```
return podList.filter((pod) => pod.includes('svclb'));
```

**Review by undsoft — CHANGES_REQUESTED** [review]
It looks good, but let's also wait for Waqar's approval.

**Comment by Ornias1993** [comment]
> It looks good, but let's also wait for Waqar's approval.

Thanks for your feedback!
I've removed the function and directly filtered the podnames like you showed. Also added a comment to make clear why it's done.

I've passed this past Waqar the 06-sept  and got no objection (while discussing https://github.com/truenas/middleware/pull/7481 ) :)

**Comment by sonicaj** [comment]
@Ornias1993 this is not required as middleware is not showing the pods spawned from daemonsets.

**Pull request closed without merging**
